nni/retiarii/converter/graph_gen.py

```diff
@@ -654,8 +641,212 @@ def convert_module(self, script_module, module, module_name, ir_model):
 
         return ir_graph, {}
 
+    def convert_module(self, script_module, module, module_name, ir_model):
+        """
+        Convert a module to its graph ir (i.e., Graph) along with its input arguments
+
+        Parameters
+        ----------
+        script_module : torch.jit.RecursiveScriptModule
+            the script module of ```module``` obtained with torch.jit.script
+        module : nn.Module
+            the targeted module instance
+        module_name : str
+            the constructed name space of ```module```
+        ir_model : Model
+            the whole graph ir
 
-def convert_to_graph(script_module, module):
+        Returns
+        -------
+        Graph
+            the built graph ir from module, ```None``` means do not further parse the module
+        dict
+            the input arguments of this module
+        """
```

Comment:
why wrap `_convert_module` with exactly the same input arguments? and seems `convert_module` does not have returns, but there are returns in the docstring